### DISCUSSION TITLE
Container image scans service reported unsupported os for watchtower

### DISCUSSION THREAD

**Discussion by AnandPalani92** [containrrr/watchtower#615] [Q&A] [answered]
I have used watchtower images for image update, Our container image scanner service reported unsupported os for watchtower image.

<img width="924" alt="Screenshot 1942-05-18 at 8 47 39 AM" src="https://user-images.githubusercontent.com/15903560/89724177-4719c780-da1d-11ea-9e5a-b37103d2767b.png">

**Comment by simonaronsson** [comment, marked as answer]
Thank you for the report!

What is this image scanner service and what does unsupported OS mean in this context? Watchtower runs on a scratch image, so there really isn't much OS to speak of, might this be why you get this message?
